.github/workflows/release.yml

```
@@ -19,3 +19,39 @@ jobs:
         env:
           NPM_TOKEN: ${{secrets.npm_token}}
       - run: pnpm publish -r --filter '!@graphql-debugger/landing-page-app' --filter '!@graphql-debugger/landing-page-server' --no-git-checks
+  #TODO
+  #Since we're already building the image in the quality testing, we should try to reuse that.
+  #
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_LOGIN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+        with: #TODO This needs to be variables
+          images: mediocredevops/graphql-debugger
+
+      #TODO
+      #Re-use artifacts from existing jobs
+      #https://docs.github.com/actions/using-workflows/storing-workflow-data-as-artifacts
+      #
+      #Use tags https://github.com/docker/metadata-action
+      - name: Build and push Docker image
+        uses: docker/build-push-action@3b5e8027fcad23fda98b2e3ac259d8d67585f671
+        with:
+          context: .
+          file: ./docker/Dockerfile
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
```

See here for npm we use the git tag as the version number: 

* https://github.com/rocket-connect/graphql-debugger/blob/main/.github/workflows/release.yml#L16
```suggestion
          tags: ${{ github.ref_name }}
```

Not sure if `steps.meta.outputs.tags` refers to the same thing?

---

Yes, the latest tag is 0.0.0-alpha.77

---

sweet that helps me a lot since the docker one is a bit confusing but it comes with things like multiple tags
```
tags: |
            type=ref,event=branch
            type=ref,event=pr
            type=semver,pattern={{version}}
            type=semver,pattern={{major}}.{{minor}}
```
So we can just give it multiple tags, I'll try both the ways.